.github/workflows/docs-workflow.yml

```diff
@@ -95,3 +95,21 @@ jobs:
         run: |
           git status
           git diff-index --quiet HEAD
+
+  # Fail if there are de-referenced links in the repository
```

Comment:
I would understand "de-reference" to mean replace by the actual file, which is not what is checked here. Can we simply call them "broken symlinks" (here and below)?

Reply:
Done